apps/api/src/newrelic.ts

```diff
@@ -29,13 +31,18 @@ exports.config = {
      */
     enabled: true,
   },
+  application_logging: {
+    forwarding: {
+      enabled: true,
+    },
+  },
   logging: {
     /**
      * Level at which to log. 'trace' is most useful to New Relic when diagnosing
      * issues with the agent, 'info' and higher will impose the least overhead on
      * production applications.
      */
-    level: 'info',
+    level: 'trace',
```

Comment:
They are mentioning that trace should be used when diagnosing things, and that higher trace levels should be used. What are the performance implications? Should we do here log and above here?